docs/configuring-playbook-collabora.md

```diff
@@ -0,0 +1,131 @@
+# Setting up Collabora integration
+
+**WARNING**: Work in progress - Currently only external reverse proxy supportet.
+
+The following configuration enables you to install the [Collabora Online Development Edition (CODE)](https://www.collaboraoffice.com/) document server.
+You will have a fully functioning [WOIP server]() that (together with the [Nextcloud Office App](https://apps.nextcloud.com/apps/richdocuments)) enables you to
+edit office documents online. This version is more performant than the [CODE App](https://apps.nextcloud.com/apps/richdocumentscode).
+
+The document server will run under a different domain than your Nextcloud.
+
+## DNS
+
+Make sure to have set the following DNS record as the document server will run under a seperate domain
+
+
+| Type  | Host        | Priority | Weight | Port | Target                     |
+|-------|-------------|----------|--------|------|----------------------------|
+| CNAME | `collabora` | -        | -      | -    | `nextcloud.yourdomain.org` |
+
+## `vars.yml` configuration
+
+Enable the server like this in your configuration file (`inventory/host_vars/<your-domain>/vars.yml`) and set a password
+for the admin interface.
+
+```yaml
+nextcloud_collabora_document_server_enabled: true
+nextcloud_collabora_document_server_password: 'lHAbqkYapmelxLWFqjrYS3v9RQtIzQbWrvs'
+```
+
+You probably should also adjust the domain name to match the DNS record (the default value is `collabora.{{ nextcloud_server_fqn_nextcloud }}`)
+
+```yaml
+nextcloud_collabora_document_server_domain: "collabora.yourdomain.org"
+```
+
+If you had already installed Nextcloud, you'll need to re-run the [installation](installing.md) procedure and restart the services.
+
+In any case, once Nextcloud is fully installed (**don't forget** that there are some manual steps [after installing](installing.md) with the playbook),
+you can also install and configure the [Nextcloud Office app](https://apps.nextcloud.com/apps/richdocuments) by running this command:
+
+```bash
+ansible-playbook -i inventory/hosts setup.yml --tags=setup-collabora-app
+```
+
+The above only does basic configuration, hooking your Collabora document server with Nextcloud.
+The Nextcloud office app supports additional options, which you can configure manually from: **Administration Settings** -> **Office**.
+
+For security reasons you should set the `Allow list for WOPI requests` to match the IP of your server. Read more under [here](https://docs.nextcloud.com/server/latest/admin_manual/office/configuration.html#wopi-settings)
+
+
+## Reverse Proxy Configuration
+
+If you use an external reverse proxy you have to configure it correctly. Here is an example of a working nginx configuration.
```

Comment:
If you're using nginx, why not use the playbook to manage the configuration for you, so you can just include it into your own nginx webserver?

See the **Using nextcloud-nginx-proxy to generate configuration for your own, other, nginx proxy** section in `docs/configuring-playbook-reverse-proxy.md`

This should make things easier for people using nginx (including you) and should simplify this documentation page.

---

Besides this, we should also add Traefik support, as that's the default for this playbook and a much better way to reverse-proxy to containers, especially when running various web-based services on the same host.

Reply:
I gave it a try but I have a hard time to wrap my head around it. If I understand correctly we would aim the default behavior to flow `Traefik`->`nextcloud-collabora-online` and an alternative as `nextcloud-nginx-proxy` -> `collabora-online` and `my reverse proxy` -> `collabora-online` (which is the case for me so I'd use `nextcloud_nginx_proxy_service_reach_mode: local` to use the configurationn generated  by nextcloud-nginx-proxy in my own nginx right?